logdb/logdb_test.go

```diff
@@ -19,6 +22,12 @@ import (
 	"github.com/vechain/thor/v2/tx"
 )
 
+var dbPath string
+
+func init() {
+	flag.StringVar(&dbPath, "dbPath", "", "Path to the database file")
+}
```

Comment:
Is this the standard way of passing args to tests ?

Reply:
Yes as far as I understand. Or we can use environment variables.